README.md

```diff
@@ -36,3 +36,10 @@ Master [git repository](http://github.com/informatikr/hedis):
 
 This library is written by Falko Peters <falko.peters@gmail.com>.
 Currently maintainer by Kostiantyn Rybnikov <k-bx@k-bx.com>.
+
+# How to Start Redis Server Using Flakes.
```

Comment:
It could follow this pattern:
```
Getting started
  -- Development
      -- Services
          -- <your description goes here>
      -- DevShell
          -- nix develop
  -- Build
     -- nix build
```

In a separate PR you can add a tiny example project and also mention how you can use that to test redis/redis-cluster.